package.json

```diff
@@ -10,7 +10,7 @@
   "homepage": "https://sauce.raspberrypi.org",
   "repository": "github:RaspberryPiFoundation/sauce-design-system",
   "bugs": "https://github.com/RaspberryPiFoundation/sauce-design-system/issues",
-  "version": "1.0.0-alpha.0",
+  "version": "1.0.0-alpha.1",
```

Comment:
One more tiny thing - this version number also appears in package-lock.json, so you'll need to generate it to make that change too.